varname/ignore.py

```diff
@@ -133,8 +145,8 @@ def __init__(self, ignore: FunctionType) -> None:
                 #'<locals>' in ignore.__qualname__ or # without functools.wraps
                 ignore.__name__ != ignore.__code__.co_name
         ):
-            warnings.warn('A decorated function might be used '
-                          'as an ignore element for varname.')
+            warnings.warn('A decorated function may be used '
+                          'as an ignore element directly for varname.')
```

Comment:
Suggested text:

> You asked varname to ignore the function `{ignore.__name__}` but it looks like this is actually a decorated function, e.g. a wrapper. If this is intended, <instructions to ignore the warning>. Otherwise, <instructions to ignore the actual function>.

Ideally there should be an easy way to suppress the warning. Simplest thing you can do is use a custom warning class.